src/athesis/kernels_pressure_equation.jl

```diff
@@ -1,19 +1,18 @@
 # kernels_pressure_equation.jl
 
-include("operators.jl")
 
 function averageK(K, i, j, k)
-    Kw = harmonic_mean(K[i-1,j  ,k  ],K[i  ,j  ,k  ])
-    Ke = harmonic_mean(K[i  ,j  ,k  ],K[i+1,j  ,k  ])
-    Ks = harmonic_mean(K[i  ,j-1,k  ],K[i  ,j  ,k  ])
-    Kn = harmonic_mean(K[i  ,j  ,k  ],K[i  ,j+1,k  ])
-    Kb = harmonic_mean(K[i  ,j  ,k-1],K[i  ,j  ,k  ])
-    Kt = harmonic_mean(K[i  ,j  ,k  ],K[i  ,j  ,k+1])
+    Kw = harmonic_mean(K[i - 1,j  ,k  ], K[i  ,j  ,k  ])

```

Comment:
Could we somehow suppress this formatting of indices? It doesn't seem to improve the readability in this case, wouldn't you agree?